examples/design/EU-DEMO/EUDEMO_builders/ivc/plasma_face.py

```diff
@@ -0,0 +1,136 @@
+# reactors. It incorporates several modules, some of which rely on other
+# codes, to carry out a range of typical conceptual fusion reactor design
+# activities.
+#
+# Copyright (C) 2021 M. Coleman, J. Cook, F. Franza, I.A. Maione, S. McIntosh, J. Morris,
+#                    D. Short
+#
+# bluemira is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# bluemira is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with bluemira; if not, see <https://www.gnu.org/licenses/>.
+"""
+Plasma Face Designer
+"""
+from dataclasses import dataclass
+from typing import Dict, Tuple
+
+import numpy as np
+
+from bluemira.base.designer import Designer
+from bluemira.base.parameter_frame import NewParameter as Parameter
+from bluemira.base.parameter_frame import NewParameterFrame as ParameterFrame
+from bluemira.geometry.face import BluemiraFace
+from bluemira.geometry.tools import boolean_cut, make_polygon
+from bluemira.geometry.wire import BluemiraWire
+
+
+@dataclass
+class PlasmaFaceDesignerParams(ParameterFrame):
+    """Parameters for running the `PlasmaFaceDesigner`."""
+
+    div_type: Parameter[str]
+    c_rm: Parameter[float]
+
+
+class PlasmaFaceDesigner(Designer[BluemiraFace]):
+    """
+    Designs the Plasma facing keep out zones
+
+    Parameters
+    ----------
+    params: Dict[str, ParameterFrame]
+        Plasma face designer parameters
+    ivc_boundary: BluemiraWire
+        IVC boundary keep out zone
+    wall_boundary: BluemiraWire
+        wall boundary keep out zone (cut at divertor)
+    divertor_silhouette: BluemiraWire
+        divertor keep out zone
+    """
+
+    param_cls = PlasmaFaceDesignerParams
+
+    def __init__(
+        self,
+        params: Dict[str, ParameterFrame],
+        ivc_boundary: BluemiraWire,
+        wall_boundary: BluemiraWire,
+        divertor_silhouette: BluemiraWire,
+    ):
+        super().__init__(params)
+        if self.params.div_type.value == "DN":
+            raise NotImplementedError("Double Null divertor not implemented")
+        self.ivc_boundary = ivc_boundary
+        self.wall_boundary = wall_boundary
+        self.divertor_silhouette = divertor_silhouette
+
+    def run(self) -> Tuple[BluemiraFace]:
+        """
+        Run method for PlasmaFaceDesigner
+        """
+        # For double null this and self.divertor_silhouette need a structure
+        # to accomodate two divertors
+        plasma_facing_wire = BluemiraWire(
+            [self.wall_boundary, *self.divertor_silhouette]
```

Comment:
This may well be code leftover from before - but I can't quite work this out. What is being unpacked here? Isn't `self.divertor_silhouette` a `BluemiraWire`?

Reply:
the output of `divertor_silhouette` is currently a tuple of wires, see L152 of `divertor_silhouette.py`. I haven't checked whether I can combine it into a single wire and still extract the individual sections (although I could have a look)